.krew.yaml

```diff
@@ -5,15 +5,20 @@ metadata:
 spec:
   version: {{ .TagName }}
   homepage: https://github.com/Azure/kubectl-aks
-  shortDescription: Interact with an AKS cluster
+  shortDescription: Interact with and debug AKS clusters
```

Comment:
Is the shortDescription different from what we have [krew](https://github.com/kubernetes-sigs/krew-index/blob/master/plugins/aks.yaml#L8) on purpose?

Reply:
Okay Perfect.